README.md

```diff
@@ -0,0 +1,29 @@
+# eslint-config-sparkbox
+
+✅ A set of eslint customizations that we use at Sparkbox. This config extends the [Airbnb config] and includes rule support for:
+
+* [React](https://github.com/yannickcr/eslint-plugin-react)
+* [Flowtype](https://github.com/gajus/eslint-plugin-flowtype)
+* [JSX a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y)
+* [ES6+ imports](https://github.com/benmosher/eslint-plugin-import)
+
+## Usage
+
+`eslint-config-sparkbox` is designed as an [ESLint Sharable Config]
+
+`npm install --save-dev eslint-config-sparkbox` or `yarn add eslint-config-sparkbox`
```

Comment:
Since we have a yarn.lock, should we recommend `yarn add` first and `npm i` secondly?